### PR TITLE
JP-1171:  New dark level2a regression test

### PR DESCRIPTION
**Pull request by dmggh**
The input and truth files have been uploaded to artifactory and the local test run looks fine.

### CODE REVIEW

File: jwst/regtest/test_dark.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+from jwst.pipeline.collect_pipeline_cfgs import collect_pipeline_cfgs
+from jwst.stpipe import Step
+
+@pytest.mark.bigdata
+def test_miri_dark_cal( _jail, rtdata, fitsdiff_default_kwargs ):
+    rtdata.get_data("miri/image/jw00001001001_01101_00001_MIRIMAGE_uncal.fits")
+
+    collect_pipeline_cfgs("config")
+    args = ["config/calwebb_dark.cfg", rtdata.input,
+            "--steps.rscd.save_results=True"]
```
Comment (hbushouse) [inline jwst/regtest/test_dark.py]: I see the results of the ``rscd`` step being saved here, but I don't see them being used anywhere, such as in the comparison of outputs. It looks like the only comparison being done is on the main output of the pipeline, which is the _dark product.

File: jwst/regtest/test_dark.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    Step.from_cmdline( args )
+    rtdata.output = "jw00001001001_01101_00001_MIRIMAGE_dark.fits"
+
+    rtdata.get_truth("truth/test_miri_dark_pipeline/\
+        jw00001001001_01101_00001_MIRIMAGE_dark.fits")
+
+    diff = FITSDiff( rtdata.output, rtdata.truth, **fitsdiff_default_kwargs )
+    assert diff.identical, diff.report
```
Comment (hbushouse) [inline jwst/regtest/test_dark.py]: Need empty parens at the end, i.e. "diff.report()"

**Comment by jdavies-st** [comment]
Ran the tests.  They pass.
```
$ pytest --bigdata --basetemp=/Users/jdavies/scratch/ jwst/regtest/ -k test_miri_dark -v --env=newdev
============================== test session starts ==============================
platform darwin -- Python 3.7.5, pytest-5.3.1, py-1.8.0, pluggy-0.13.1 -- 
cachedir: .pytest_cache
rootdir: /Users/jdavies/dev/jwst, inifile: setup.cfg
plugins: asdf-2.4.3a1.dev13+gc1373db, xdist-1.30.0, requests-mock-1.7.0,
openfiles-0.4.0, ci-watson-0.4, forked-1.1.3, doctestplus-0.5.0, cov-2.8.1
collected 8 items / 7 deselected / 1 selected                                                                                                         

jwst/regtest/test_dark.py::test_miri_dark_pipeline PASSED                  [100%]

================== 1 passed, 7 deselected in 157.10s (0:02:37) ==================
```

**Comment by hbushouse** [comment]
@jdavies-st  Why does your test run show 8 items collected, with 7 deselected? It looks to me like this code module should only generate 1 test.

**Comment by jdavies-st** [comment]
Pytest collected 8 tests (in this branch) in the `jwst/regtest` dir, then the `-k test_miri_dark` deselected 7 of those because they did not match that string.

**Pull request merged**